schema/validator.go

```diff
@@ -41,7 +41,7 @@ func (e ValidationError) Error() string {
 func (v Validator) Validate(src io.Reader) error {
 	buf, err := ioutil.ReadAll(src)
 	if err != nil {
-		return errors.Wrap(err, "unable to read manifest")
+		return errors.Wrap(err, "unable to read the <schema.json> file")
```

Comment:
The *schema* is being loaded below (`NewReferenceLoaderFileSystem("file:///"+specs[v], fs)`).  This error is about the document being validated.